core/frontend/src/internal/render/webgl/BranchUniforms.ts

```diff
@@ -267,4 +269,11 @@ export class BranchUniforms {
       this.clipStack.intersectionStyle.alpha = disableClipStyle ? 0 : (style.intersectionStyle ? style.intersectionStyle.width : 0);
     }
   }
+
+  private setContourLine(contourLine: ContourDisplay) {
```

Comment:
No, you are still allowing the Target to alter the Viewport, and you are assuming you are drawing to the ViewManager's selected view, which will not always be true.